airflow/bin/cli.py

```diff
@@ -1066,6 +1066,10 @@ def flower(args):
     if args.broker_api:
         api = '--broker_api=' + args.broker_api
 
+    url_prefix = ''
+    if args.url_prefix:
```

Comment:
I wonder if this should be args.flower_url_prefix?

Reply:
all variables in this methods are related to flower (see some lines above, `api` is not `flower_api`)